src/ResourceManager/Insights/Commands.Insights/Metrics/GetAzureRmMetricCommand.cs

```diff
@@ -127,16 +129,22 @@ protected string ProcessParameters()
         /// </summary>
         protected override void ProcessRecordInternal()
         {
+            WriteWarning("API deprecation: The use of the legacy metrics API will be discontinued in the next release. This implies a change in the call and the output of this cmdlet.");
+
             string queryFilter = this.ProcessParameters();
             bool fullDetails = this.DetailedOutput.IsPresent;
 
-            // Call the proper API methods to return a list of raw records.
-            // If fullDetails is present full details of the records displayed, otherwise only a summary of the values is displayed
-            MetricListResponse response = this.InsightsClient.MetricOperations.GetMetricsAsync(resourceUri: this.ResourceId, filterString: queryFilter, cancellationToken: CancellationToken.None).Result;
```

Comment:
No, response is still a MetricListResponse.

Here I am consuming the old SDK code, that I just added to the project, instead of the code in the new SDK library.
I had to add the using block around the call to create an old client and be able to call the exact same method I used to call in the previous (SDK).
This change is only to avoid a breaking change. The call and the output remain identical between the previous and this release.